src/api/apiErrors.js

```diff
@@ -168,3 +169,25 @@ export const interpretApiResponse = (httpStatus: number, data: mixed): mixed =>
   // the API says that shouldn't happen.
   throw new UnexpectedHttpStatusError(httpStatus, data);
 };
+
+/**
+ * The Zulip Server version below which we should just refuse to connect.
+ */
+// Currently chosen to affect a truly tiny fraction of users, as we test the
+// feature of refusing to connect, to keep the risk small; see
+//   https://github.com/zulip/zulip-mobile/issues/5102#issuecomment-1233446360
+// In steady state, this should lag a bit behind the threshold version for
+// ServerCompatBanner, to give users time to see and act on the banner.
+export const kThresholdZulipVersion: ZulipVersion = new ZulipVersion('2.0');
```

Comment:
This name feels pretty non-specific to me — it says this is a version where we make some kind of distinction between above and below it, but it doesn't really say what that distinction is.

Tempted to say `kMinZulipVersion`, but I guess there's already a `kMinSupportedVersion` with a different meaning, so it's better to draw more specifically the contrast from that one.

How about `kMinAllowedServerVersion`?